package-testing/ruby-sdk-relay/src/server.rb

```diff
@@ -0,0 +1,133 @@
+require 'bundler/setup'
+require 'sinatra'
+require 'json'
+require 'eppo_client'
+
+class LocalAssignmentLogger < EppoClient::AssignmentLogger
+  def log_assignment(assignment)
+    puts "Assignment: #{assignment}"
+  end
+end
+
+def initialize_client_and_wait
+  puts "Initializing client"
+  api_key = ENV['EPPO_API_KEY'] || 'NOKEYSPECIFIED'
+  base_url = ENV['EPPO_BASE_URL'] || 'http://localhost:5000/api'
+  
+  client_config = EppoClient::Config.new(
+    api_key,
+    base_url: base_url,
+    assignment_logger: LocalAssignmentLogger.new
+  )
+  
+  EppoClient::init(client_config)
+  client = EppoClient::Client.instance
+  sleep(3)
+  puts "Client initialized"
+end
+
+# Health check endpoint
+get '/' do
+  "OK"
+end
+
+# SDK reset endpoint
+post '/sdk/reset' do
+  initialize_client_and_wait
+  "Reset complete"
+end
+
+# SDK details endpoint
+get '/sdk/details' do
+  content_type :json
+  {
+    sdkName: "ruby-sdk",
+    sdkVersion: "3.3.0",
+    supportsBandits: false,
+    supportsDynamicTyping: false
+  }.to_json
+end
+
+# Assignment endpoint
+post '/flags/v1/assignment' do
+  content_type :json
+  data = JSON.parse(request.body.read)
+  
+  client = EppoClient::Client.instance
+  
+  begin
+    result = case data['assignmentType']
```

Comment:
Ruby looks pretty slick. I like this select-expression-return. Kotlin has this as well.

Reply:
_Ruby_ is slick*. Ruby on rails is an abomination.

*except for the `unless` keyword, which is also an abomination